cachito/workers/pkg_managers/general_js.py

```diff
@@ -34,6 +36,27 @@
 log = logging.getLogger(__name__)
 
 
+def get_git_prefixes():
```

Comment:
Optional: I know this sounds a little obsessive, but the whole Python module is alphabetized so it'd be nice to keep this consistent. :stuck_out_tongue: